client/lib/abtest/index.js

```diff
@@ -96,7 +107,17 @@ ABTest.prototype.init = function( name ) {
 	this.defaultVariation = testConfig.defaultVariation;
 	this.variationNames = variationNames;
 	this.experimentId = name + '_' + variationDatestamp;
-	this.allowAnyLocale = testConfig.allowAnyLocale === true;
+
+	this.localeTargets = [ 'en' ];
+
+	if ( 'undefined' !== typeof testConfig.localeTargets ) {
+		if ( testConfig.localeTargets === false ) {
+			this.localeTargets = false;
+		} else {
+			this.localeTargets = testConfig.localeTargets;
+		}
+	}
```

Comment:
How do you feel about swapping in the string `'any'` for the behaviors you're triggering right now with `false`?  

Two reasons I ask:
1. Setting `testConfig.localeTargets = 'any'` feels relatively intuitive.
2. Then you can do something much simpler here like:
`this.localeTargets = testConfig.localeTargets || [ 'en' ];`